androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -261,4 +280,11 @@ public void onServiceDisconnected(ComponentName componentName) {
             mSession = null;
         }
     }
+
+    public interface FallbackStrategy {
```

Comment:
nit: could you move the interface to the top (after private variables).

Reply:
done